source/fundamentals/crud/compound-operations.txt

```diff
@@ -0,0 +1,313 @@
+===================
+Compound Operations
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide you will learn how to perform **compound operations** with the
+MongoDB Java driver. 
+
+Compound operations consist of a read and write operation performed as one
+**atomic operation**. An atomic operation is an operation which either completes
+entirely, or does not complete at all. Atomic operations cannot partially complete.
+
+Atomic operations can help you avoid **race conditions** in your code. A
+race condition occurs when your code's behavior is dependent on the order of
+uncontrollable events.
+
+MongoDB supports the following compound operations:
+
+- Find and update one document
+- Find and replace one document
+- Find and delete one document
```

Comment:
This seems a bit out of place in the overview. I think this should go into your How to Use Compound Operations section and if you want, you can hyperlink each one to their respective sections.

Reply:
I disagree, and feel that this information is necessary for a reader to determine if this is the page they are looking for. Without seeing the complete list of supported compound operators, I do not think a reader will be able to make a judgment call on whether they should read the compound ops page, the transactions page, or a different CRUD page.

Reply:
This comment was discussed synchronously and it was agreed that this information is necessary for a reader to make a judgment call if they are on the correct page.